### PR TITLE
Refactoring functions

### PR DESCRIPTION
**Pull request by cescgrau**

### CODE REVIEW

File: domain/fizz_buzz/extended.py [hunk 1]
```diff
@@ -1,16 +1,15 @@
-from domain.fizz_buzz.tools import is_multiple_of_five, is_multiple_of_three, is_number_three_into_the_number, \
-    is_number_five_into_the_number
+from domain.fizz_buzz.tools import is_multiple_of, is_number_a_into
 
 
 def map_from_integer_to_fizz_and_buzz_extended(a_number):
     accumulated_result = ""
-    if is_number_three_into_the_number(a_number):
+    if is_number_a_into(character="3", a_number=a_number):
```
Comment (jvalduvieco) [inline domain/fizz_buzz/extended.py]:
```suggestion
    if is_character_into(character="3", a_number=a_number):
```

File: domain/fizz_buzz/tools.py [hunk 2]
```diff
@@ -6,15 +6,24 @@ def is_multiple_of_three(a_number):
     return (a_number % 3) == 0
 
 
+def is_multiple_of(multiple, a_number):
+    return (a_number % multiple) == 0
```
Comment (jvalduvieco) [inline domain/fizz_buzz/tools.py]: No need for parentheses. See: https://www.programiz.com/python-programming/precedence-associativity

File: test/domain/fizz_buzz/test_fizz_buzz_tools.py [hunk 3]
```diff
@@ -33,6 +26,14 @@ def test_should_return_true_if_an_integer_contains_five(self):
     def test_should_return_true_if_an_integer_not_contains_five(self):
         self.assertFalse(is_number_five_into_the_number(44))
 
+    def test_contains_a(self):
```
Comment (jvalduvieco) [inline test/domain/fizz_buzz/test_fizz_buzz_tools.py]: Try to be more explicit about the expectation in the test name. Something like:
test_if_a_number_contains_a_given_character_return_true

File: test/domain/fizz_buzz/test_fizz_buzz_tools.py [hunk 4]
```diff
@@ -1,25 +1,18 @@
...
-        self.assertFalse(an_integer_result)
-
-    def test_an_integer_multiple_of_three_is_identified(self):
-        an_integer_result = is_multiple_of_three(9)
-        self.assertTrue(an_integer_result)
-
-    def test_an_integer_not_multiple_of_three_is_identified(self):
-        an_integer_result = is_multiple_of_three(8)
-        self.assertFalse(an_integer_result)
+    def test_an_integer_is_multiple_of(self):
```
Comment (jvalduvieco) [inline test/domain/fizz_buzz/test_fizz_buzz_tools.py]: Try to be more explicit about the expectation in the test name.

**Comment by jvalduvieco** [comment]
No dead code to be deleted? :thinking:

File: domain/fizz_buzz/tools.py (hunk 2 above)
Comment (cescgrau) [reply domain/fizz_buzz/tools.py]: ok, clar. els he tret també.

File: test/domain/fizz_buzz/test_fizz_buzz_tools.py (hunk 4 above)
Comment (cescgrau) [reply test/domain/fizz_buzz/test_fizz_buzz_tools.py]: Wired, I can't see this file, even the test_fizz_buzz_tools.py is not on this folder. It's in the Test folder.

**Review by jvalduvieco — APPROVED** [review]

**Pull request merged**